harness/determined/_core/_searcher.py

```diff
@@ -10,57 +10,38 @@
 
 
 class Unit(enum.Enum):
-    EPOCHS = "UNIT_EPOCHS"
-    RECORDS = "UNIT_RECORDS"
-    BATCHES = "UNIT_BATCHES"
+    EPOCHS = "EPOCHS"
+    RECORDS = "RECORDS"
+    BATCHES = "BATCHES"
+
+
+def _parse_searcher_units(experiment_config: dict) -> Optional[Unit]:
```

Comment:
this is all a bit unfortunate, that there isn't nicer tooling to get something so simple

Reply:
I don't think it's that simple.  The concept of "which units does the searcher use" isn't configured as `searcher.units`, it's inferred from other searcher properties.  At least IMO, the issue is with the complexity of the config and not with the tooling.